tests/suites/spaces_ec2/juju_bind.sh

```diff
@@ -9,8 +9,10 @@ run_juju_bind() {
     juju reload-spaces
     juju add-space isolated 172.31.254.0/24
 
-    ## Create machine
-    add_multi_nic_machine
+    # Note that due to the way that run_* funcs are executed, $1 holds the
+    # test name so the NIC ID is actually provided in $2
+    hotplug_nic_id=$2
```

Comment:
Did we not want to fix this?

Reply:
Yes but I thought it was less effort to simply follow the same pattern as the other tests which we already know that work as expected. Perhaps we can do this via a separate PR.